src/inpage-panel.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8"/>
+    <title>Firefox Relay</title>
+    <link rel="stylesheet" href="/css/relay.css" />
+    <link rel="stylesheet" href = "/css/first-run.css" />
+    <script src="/js/libs/browser-polyfill.min.js"></script>
+    <script src="/js/shared/utils.js"></script>
+    <script src="/js/shared/metrics.js"></script>
+    <script src="/js/other-websites/fathom.js"></script>
+    <script src="/js/other-websites/email_detector.js"></script>
+    <script src="/js/other-websites/inpage_menu.js"></script>
+    <script src="/js/other-websites/fill_relay_address.js"></script>
+  </head>
+  <body class="fx-relay-menu-body">
+    <div>
+        <div class="fx-relay-menu">
+
+          <!-- Logged In Content -->
+          <div class="fx-content-signed-in">
+            <div class="fx-relay-alias-loading-image"><img /></div>
+
+            <span class="fx-relay-menu-remaining-aliases"></span>
+            <button class="fx-relay-menu-generate-alias-btn"></button>
+            
+            <a class="fx-relay-menu-get-unlimited-aliases" target="_blank" rel="noopener noreferrer" href=""></a>
+            <a class="fx-relay-menu-dashboard-link" href="" target="_blank"></a>
```

Comment:
Would filling the `href`s here cause problems? Would be nice if you could see where the link is going while hovering it. (In fact, why are we sending people there with an onclick handler in the first place?)

Reply:
We are technically setting the `href` when the link is rendered. We have a [telemetry event that we're appending](https://github.com/mozilla/fx-private-relay-add-on/blob/spike-iframe-concept/src/js/other-websites/inpage_menu.js#L168-L176). (The onClick does not prevent anything, just adds to it). 

Removed the empty `href` attribute in ed515009a70c4fd3b452eff9955d37539efb41bc